learn/educators/python.md

```diff
@@ -0,0 +1,98 @@
+---
+Order: 4
+Area: educators
+TOCTitle: Python
+ContentId: e58523d5-e243-45c2-8306-7114077d81ae
+PageTitle: Python in Visual Studio Code
+DateApproved: 02/17/2021
+MetaDescription: Python in Visual Studio Code
+---
+
+# Python
+
+Visual Studio Code is free professional editor that fully supports Python and other features like real-time collaboration. It's highly customizable to support your classroom the way you like to teach.
+
+> "[Visual Studio Code] is the best balance of authenticity and accessibility. Plus, [Visual Studio Code] works on every OS!" - Professor Zachary Dodds from Harvey Mudd College
+
+Read below for recommendations for extensions, settings, and links to free lessons that you can use in your classes.
+
+## Python Extension Pack for the classroom
+
+Unsure of which extensions to recommend to your students for your class? You can point your students to the [Python Education Extension Pack](https://marketplace.visualstudio.com/items?itemName=tanhakabir.python-education-extension-pack) that contains all of the essential and helpful extensions for the classroom. You can download the pack from the Extension Marketplace:
+
+![Python Education Extension Pack](images/python/python-extension-pack.png)
+
+The pack contains:
+* [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.python) for basic Python functionality like compiling, debugging support, linting, Jupyter Notebooks, unit tests, and more.
+* [Live Share](https://marketplace.visualstudio.com/items?itemName=MS-vsliveshare.vsliveshare-pack) to enable real-time collaboration and [Live Share Audio](https://marketplace.visualstudio.com/items?itemName=MS-vsliveshare.vsliveshare-audio) to enable audio calls as well.
+* [Remote - SSH](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh) to work on remote projects (like ones that live on lab machines) through SSH with full Visual Studio Code functionality.
+* [Markdown+Math](https://marketplace.visualstudio.com/items?itemName=goessner.mdmath) for full Latex support in Markdown
+* [GitLens](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens) to enhance Git capabilities on Visual Studio Code like inline Git blame.
+
+## How CS5 from Harvey Mudd College uses Visual Studio Code to teach Python
```

Comment:
These H2s are pretty long and will word wrap on the right navigation list.
What about just "Harvey Mudd case study" or "Use at Harvey Mudd"?